core/encoding/base64/base64.odin

```diff
@@ -90,4 +90,39 @@ decode :: proc(data: string, DEC_TBL := DEC_TABLE, allocator := context.allocato
         out[j + 2] = byte(b2);
     }
     return out;
-}
\ No newline at end of file
+}
+
+// @note(zh): Test inputs. Taken from RFC4648
+/*
+import "core:fmt"
+main :: proc() {
+    Test :: struct {
+        plain: string,
+        encoded: string,
+    };
+
+    test_vectors := [?]Test {
+        Test{"", ""},
+        Test{"f", "Zg=="},
+        Test{"fo", "Zm8="},
+        Test{"foo", "Zm9v"},
+        Test{"foob", "Zm9vYg=="},
+        Test{"fooba", "Zm9vYmE="},
+        Test{"foobar", "Zm9vYmFy"},
+    };
+
+    // Encode test
+    for v in test_vectors {
+        enc := encode(([]byte)(v.plain));
+        fmt.printf("encode(\"%s\") => \"%s\" \t| want: \"%s\"\n", v.plain, enc, v.encoded);
+        delete(enc);
+    }
+
+    // Decode test
+    for v in test_vectors {
+        dec := decode(v.encoded);
+        fmt.printf("decode(\"%s\") => \"%s\" \t| want: \"%s\"\n", v.encoded, string(dec), v.plain);
+        delete(dec);
+    }
+}
```

Comment:
These probably shouldn't stick around in the actual `core` library

Reply:
I can remove them. I had them for proof initially, since those are the test vectors from the actual RFC

Reply:
Hmm, you could also turn it into a usage example as you have in `csv` and then do that same for `base32`?